RNWCPP/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -306,8 +346,47 @@ void ViewViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate, folly::dyna
       }
       else if (propertyName == "acceptsKeyboardFocus")
       {
-        if (pair.second.isBool())
-          shouldBeControl = pair.second.asBool();
+        if (propertyValue.isBool())
+          shouldBeControl = propertyValue.asBool();
+      }
+      else if (propertyName == "accessibilityRole")
+      {
+        // FUTURE having an accessibilityRole shouldBeControl to support
+        // non-Touchable scenarios
+        if (propertyValue.isString())
+        {
+          auto role = propertyValue.asString();
+          if (role == "none")
```

Comment:
If the role isn't one we support then this code will leave it at the previous value. Is that the right thing to do or should we set it to None?

Reply:
i'll add else do none